vgscollect/src/main/java/com/verygoodsecurity/vgscollect/util/extension/String.kt

```diff
@@ -67,3 +66,20 @@ internal fun String.substringOrNull(startIndex: Int, endIndex: Int) = try {
 } catch (e: Exception) {
     null
 }
+
+internal fun String.applyLimitOnMask(limit: Int): String {
+    return if (limit < 0 || limit >= this.length) {
+        this
+    } else {
+        val builder = StringBuilder()
+        var tempLimit = limit
+        for (i in this.indices) {
+            val c = this[i]
+            if (c != MASK_ITEM) tempLimit += 1
```

Comment:
@Endorf Should be skipped not mask chars? I mean as the dev applies length, does he expect that this length will include all chars and divider character too?